src/components/Newsletter/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Mailchimp from "./Mailchimp";
+import styles from "./styles.module.css";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+
+const Newsletter = () => {
+  const context = useDocusaurusContext();
+  const {
+    siteConfig: {
+      themeConfig: {
+        mailchimp: { submitUrl = null },
+      },
+    },
+  } = context;
```

Comment:
Is that the right place to put the submit URL? I'm not sure what `themConfig` is normally used for.

Reply:
The theme config is used API keys, analytics tags etc. This seems to be the docusaurous convention.